restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCDevice.java

```diff
@@ -370,6 +370,10 @@ public RCConnection connect(Map<String, Object> parameters, RCConnectionListener
             connection.state = RCConnection.ConnectionState.PENDING;
             DeviceImpl.GetInstance().sipuaConnectionListener = connection;
 
+            if (enableVideo == null) {
```

Comment:
Of course I did :) That's why I added this check.
When enableVideo is null, then enableVideo.booleanValue() throws NPE:
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Boolean.booleanValue()' on a null object reference